free-2.md

```diff
@@ -11,34 +11,43 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+A lot of hardware design is done via hardware description languages (HDLs). While this is preferable to designing the logic yourself, HDLs are notoriously bad to work with and are about as intuitive as using C. It would be nice if we could make a DSL for generating hardware designs (or at least for generating HDL code). This could help hardware designers avoid some of the issues with working with HDLs and make their lives easier.
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+HDLs are already on the verge of a DSL themselves. While they need to be Turing complete as they need to build all kinds of hardware, they fairly deterministically convert certain expressions into certain logical elements. This feature is pretty DSL-y as well as the fact that the intended output is actually hardware instead of some kind of data value or file like in GPPLs. However, as stated earlier, HDLs don't do a very good job. Thus, it could be nice to create a new DSL that allows programmers to focus on what's being done or on the logical elements being created instead of doing this dance of writing code in a form that gets the compiler to do what you want with the hardware you want.
```

Comment:
This makes sense. Again, I think you'll want input from domain experts for this, but I agree that a dsl seems very appropriate.